kieserver/modules/kieserver/module.yaml

```diff
@@ -23,7 +23,7 @@ envs:
 - name: "BUSINESS_CENTRAL_DISTRIBUTION_EAP"
   value: "jboss-eap-7.4"
 - name: "JBPM_WB_KIE_SERVER_BACKEND_JAR"
-  value: "jbpm-wb-kie-server-backend-7.67.0.Final-redhat-00008.jar"
+  value: "jbpm-wb-kie-server-backend-7.67.0.Final-redhat-00015.jar"
 # remember to also update "JBPM_WB_KIE_SERVER_BACKEND_JAR" value
 artifacts:
 - name: "rhpam_kie_server_distribution.zip"
```

Comment:
Don't you also need to update md5sum for rhpam_kie_server_distribution.zip and rhpam_business_central_distribution.zip here?

Reply:
yep, fixed